gothemis/cell/cell.go

```diff
@@ -283,6 +283,9 @@ func (sc *SecureCell) Protect(data []byte, context []byte) ([]byte, []byte, erro
 	if sizeOverflow(encLen) || sizeOverflow(addLen) {
 		return nil, nil, ErrOutOfMemory
 	}
+	if sizeOverflow(encLen) || sizeOverflow(addLen) {
```

Comment:
this is wrong, it's a duplication of the lines above, but using previous error name